docs/docs/functions/macros.md

```diff
@@ -143,8 +143,126 @@ Defines a new function.
 === "Code"
 
     ```clojure linenums="1"
+    (defn fn-name [fn-parms] ;; function-name followed by input parameters in '[]'
+        (Msg parm)           ;; function body that returns a value
+    )
+    ```
+
+The function definition consists of the function name followed by its input parameters (parms), in `[]`.
+
+If there are no input parms the `[]` remains empty. Multiple input parms may be passed as a sequence.
```

Comment:
I would keep either "parameters" full word or use `fn-params` in "code" formatting. But abbreviations in text form should be avoided in my opinion.

Reply:
Agreed, fixed.